aroquev/src/main/webapp/style.css

```diff
@@ -8,7 +8,118 @@
   margin-top: 20px;
 }
 
-#main-photo {
+.body-img {
     width: 600px;
-    height: 400px;
+}
+
+/* Rules for the GitHub and LinkedIn icons */
+#github-link {
+    color: rgb(132, 0, 255);
+}
+
+#linkedin-link {
+    color: rgb(0, 81, 255);
+}
+
+#external-links img {
+    vertical-align: middle;
+    width: 50px;
+    height: 50px;
+}
+
+/* Rules for the photo gallery */
+* {box-sizing:border-box}
```

Comment:
I'll get on my soapbox a little here.

I am personally not a big fan of the global `box-sizing` reset since many people use it just so they can set `width: 100%` or `height: 100%` on something and not risk overlapping its container (would happen when there is non-zero padding & border). 

In most cases, you don't really want to explicitly set `height` on anything, and only in some cases would you want to set `width`. Normally you should let the size of the content will dictate the dimensions of the element and let everything just _flow_...

That said, if you love border-box as default, don't let me stop you.

Reply:
I am literally a noob in  CSS, but will study a bit and come back to this! Thanks!